app/frontend/src/components/MarkdownInput.tsx

```diff
@@ -11,12 +11,18 @@ const useStyles = makeStyles((theme) => ({
     "& .tui-editor-contents": {
       fontSize: theme.typography.fontSize,
       fontFamily: theme.typography.fontFamily,
-      "& h1": {
+      "& h1, & h2, & h3, & h4, & h5, & h6": {
         borderBottom: "none",
+        paddingBottom: 0,
```

Comment:
How about adding a `margin: 0` here, there are still some weird ones of a few pixels. But maybe it won't work, I don't think jss respects object order?

Reply:
Haven't time to look into properly yet (maybe tomorrow...), but the only place where that might not work is for the `first-of-type` rule I got below due to higher specificity. It's actually why I wrote that so it matches the highly specific CSS Toast UI sets on titles by default...